app/controllers/address/InternationalAddressChoiceController.scala

```diff
@@ -40,33 +40,33 @@ class InternationalAddressChoiceController @Inject() (
 )(implicit configDecorator: ConfigDecorator, templateRenderer: TemplateRenderer, ec: ExecutionContext)
     extends AddressController(authJourney, withActiveTabAction, cc, displayAddressInterstitialView) {
 
-  def onPageLoad(typ: AddrType): Action[AnyContent] =
+  def onPageLoad: Action[AnyContent] =
     authenticate.async { implicit request =>
       addressJourneyEnforcer { _ => _ =>
         cachingHelper.gettingCachedAddressPageVisitedDto { addressPageVisitedDto =>
           cachingHelper.enforceDisplayAddressPageVisited(addressPageVisitedDto) {
             Future.successful(
-              Ok(internationalAddressChoiceView(InternationalAddressChoiceDto.form(), typ))
+              Ok(internationalAddressChoiceView(InternationalAddressChoiceDto.form(), SoleAddrType))
```

Comment:
is SoleAddrType always the same? If it is, do we need to pass it through?

Reply:
There's some methods shared between change address and change postal address.
Before the regular address had 'primary' and 'sole', so we removed 'primary', but we still need it in a few places to make it distinct from 'postal'

Should be able to remove 'primary' though thinking about it

Reply:
Done